### PR TITLE
chore: add directory property in `packages.json`

### LINKED ISSUE #2474: Add `directory` to package json
Add `directory` to `repository` key in the package.json. That should help with renovate bot using a single PR. See for example: https://github.com/stryker-mutator/stryker-js/blob/78cfecce0cf3068c474ef675c7de21dacc87059b/packages/core/package.json#L18

### PR DESCRIPTION
**Pull request by xandervedder**
closes #2474

### CODE REVIEW

File: packages/elements/package.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/stryker-mutator/mutation-testing-elements.git"
+    "url": "git+https://github.com/stryker-mutator/mutation-testing-elements.git",
```
Comment (nicojs) [inline packages/elements/package.json:22]: I think we should remove the`git+` prefix while we're at it. I'm pretty sure this isn't a valid URL, which might be a problem for renovate bot.

**Review by nicojs — CHANGES_REQUESTED** [review]

File: packages/elements/package.json (hunk 1 above)
Comment (xandervedder) [reply packages/elements/package.json:22]: I also see this in the root `package.json`, should I remove this one as well?

Comment (hugo-vrijswijk) [reply packages/elements/package.json:22]: Actually renovatebot doesn't look at `repository` much for grouping PR's. It's done hardcoded: https://github.com/renovatebot/renovate/pull/21905

Comment (nicojs) [reply packages/elements/package.json:22]: This blows my mind! 🤯

Comment (nicojs) [reply packages/elements/package.json:22]: This blows my mind! 🤯

**Review by nicojs — APPROVED** [review]

**Pull request merged**